src/editor/_main.cpp

```diff
@@ -9,6 +9,8 @@
 #include "src/engine/util/util_error_handling.h"
 #include <src/engine/global.h>
 #include "editor.h"
+#include <Windows.h>
```

Comment:
what is windows.h used for?

Reply:
Removed in most recent commit. This was just me testing working dir stuff on windows for a little bit.